apps/relgen/src/index.ts

```diff
@@ -77,7 +77,7 @@ const cli = program
 
     if (config) {
       configFile = configSchema.parse(
-        JSON.parse(await readFile(config, 'utf-8'))
+        JSON.parse((await readFile(config, 'utf-8')).trim())
```

Comment:
The use of `trim` is not allowed as per the rules. Please find an alternative approach to handle any unwanted whitespace.